central/views/imagecve/view_test.go

```diff
@@ -186,8 +186,32 @@ func (s *ImageCVEViewTestSuite) TestGetImageCVECore() {
 }
 
 func (s *ImageCVEViewTestSuite) TestGetImageCVECoreSAC() {
-	for _, tc := range s.testCases() {
-		for key, sacTC := range s.sacTestCases() {
+	// Note: The new loop variable semantics introduced in go1.22 breaks these
```

Comment:
Please consider this fix instead:
https://github.com/stackrox/stackrox/compare/michael/refactor-activecomponent-updater-test..michael/refactor-imagecve-view-test